ganttproject/src/net/sourceforge/ganttproject/gui/ResourceAssignmentsTableModel.java

```diff
@@ -0,0 +1,179 @@
+package net.sourceforge.ganttproject.gui;
+
+import net.sourceforge.ganttproject.language.GanttLanguage;
+import net.sourceforge.ganttproject.resource.HumanResource;
+import net.sourceforge.ganttproject.task.ResourceAssignment;
+import net.sourceforge.ganttproject.task.ResourceAssignmentMutator;
+import net.sourceforge.ganttproject.task.Task;
+
+import javax.swing.table.AbstractTableModel;
+import java.util.*;
+
+/**
+ * @author Oleg Kushnikov
+ */
+public class ResourceAssignmentsTableModel extends AbstractTableModel {
+  enum Column {
+    ID("id", String.class),
+    NAME("taskname", String.class),
+    UNIT("unit", String.class);
```

Comment:
It's Float now with the same validating capabilities. Chose Float as float is already used as data type for `setLoad()` and `getLoad()` in `ResourceAssignment`